Python/optimizer_cases.c.h

```diff
@@ -1687,14 +1708,16 @@
         }
 
         case _EXPAND_METHOD: {
-            _Py_UopsSymbol *method;
+            _Py_UopsSymbol **method;
             _Py_UopsSymbol **self;
-            self = &stack_pointer[-1 - oparg];
-            method = sym_new_not_null(ctx);
+            method = &stack_pointer[0];
+            self = &stack_pointer[1];
+            for (int _i = 1; --_i >= 0;) {
+                method[_i] = sym_new_not_null(ctx);
+            }
```

Comment:
These stack indexes also seem off. How is `oparg` not part of the calculation? It looks like the cases generator thinks the stack has already been shrunk.

Also, maybe we should special-case length-1 arrays in the cases generator? It took me *way* too long before I realized that this whole loop was just an assignment.